metadata-service/restli-client/src/main/java/com/linkedin/common/client/BaseClient.java

```diff
@@ -41,8 +42,10 @@ protected <T> Response<T> sendClientRequest(final AbstractRequestBuilder<?, ?, ?
   @SneakyThrows
   protected <T> Response<T> sendClientRequest(
       final AbstractRequestBuilder<?, ?, ? extends Request<T>> requestBuilder,
-      @Nonnull final Authentication authentication) throws RemoteInvocationException {
-    requestBuilder.addHeader(HttpHeaders.AUTHORIZATION, authentication.getCredentials());
+      @Nullable final Authentication authentication) throws RemoteInvocationException {
+    if (authentication != null) {
```

Comment:
There is a method just above this which passes null as the second argument.

Reply:
thanks for fixing this